src/gui/editors/TrackContainerView.cpp

```diff
@@ -89,8 +89,8 @@ TrackContainerView::TrackContainerView( TrackContainer * _tc ) :
 	m_tc->setHook( this );
 	//keeps the direction of the widget, undepended on the locale
 	setLayoutDirection( Qt::LeftToRight );
-	auto layout = new QVBoxLayout(this);
-	layout->setMargin( 0 );
+	QVBoxLayout * layout = new QVBoxLayout( this );
+	layout->setContentsMargins( 0, 0, RIGHT_SCROLLBOX_MARGIN, 0 );
```

Comment:
```suggestion
	auto layout = new QVBoxLayout(this);
	layout->setContentsMargins(0, 0, RIGHT_SCROLLBOX_MARGIN, 0);
```
This doesn't need to change off `auto`, and adding whitespace is no-no. See [Coding guidelines](https://github.com/LMMS/lmms/wiki/Coding-Conventions#whitespace).

Reply:
Sorry, the `auto` change must have been when I merged my commit with some upstream commits I hadn't pulled down